target/toolchain/Makefile

```diff
@@ -58,6 +58,7 @@ $(BIN_DIR)/$(TOOLCHAIN_NAME).tar.bz2: clean
 	find $(TOOLCHAIN_BUILD_DIR) -name .git | $(XARGS) rm -rf
 	find $(TOOLCHAIN_BUILD_DIR) -name .svn | $(XARGS) rm -rf
 	find $(TOOLCHAIN_BUILD_DIR) -name CVS | $(XARGS) rm -rf
+	mkdir -p $(shell dirname $@)
```

Comment:
It would work, but it isn't directly used by tar a few lines down either. That's why I preferred to use $@ instead.
If that is preferred, I would happily accept this being amended to
```shell
mkdir -p $(BIN_DIR)
```